### ISSUE TITLE
Details view crash

### ISSUE THREAD

**Issue by moviuro** [nextcloud/android#5577]
### Steps to reproduce
1.  Open a folder
2.  On one file, tap the 3 dots
3.  Tap "Details"

### Expected behaviour
- Details view

### Actual behaviour
- Crash shortly after showing the tabbed interface (Activity | Sharing)

### Environment data
Android version: **10**

Device model: **Nokia 7 Plus**

Stock or customized system: **Stock**

Nextcloud app version: **3.10.1**

Nextcloud server version: **18.0.x** (1? unsure)

#### Web server error log
```
Probably irrelevant
```

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

javax.net.ssl.SSLHandshakeException: Handshake failed
at 
com.google.android.gms.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(:com.google.android.gms@200475037@20.04.75 
(120400-295998686):38)
at 
okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:320)
at 
okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:284)
at 
okhttp3.internal.connection.RealConnection.connect(RealConnection.java:169)
at 
okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:258)
at 
okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at 
okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at 
okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at 
okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at 
okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at 
okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:127)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at 
okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:257)
at okhttp3.RealCall.execute(RealCall.java:93)
at com.nextcloud.common.OkHttpMethodBase.execute(OkHttpMethodBase.kt:126)
at com.nextcloud.common.NextcloudClient.execute(NextcloudClient.kt:84)
at 
com.owncloud.android.lib.resources.activities.GetActivitiesRemoteOperation.run(GetActivitiesRemoteOperation.java:131)
at com.nextcloud.common.NextcloudClient.execute(NextcloudClient.kt:80)
at 
com.owncloud.android.ui.fragment.FileDetailActivitiesFragment.lambda$fetchAndSetData$5$FileDetailActivitiesFragment(FileDetailActivitiesFragment.java:331)
at 
com.owncloud.android.ui.fragment.-$$Lambda$FileDetailActivitiesFragment$N7POe6TZvoGcuV2qP-niPxY_7t4.run(Unknown 
Source:12)
at java.lang.Thread.run(Thread.java:919)
Caused by: javax.net.ssl.SSLProtocolException: SSL handshake aborted: 
ssl=0x7cce602fc8: Failure in SSL library, usually a protocol error
error:1000042e:SSL routines:OPENSSL_internal:TLSV1_ALERT_PROTOCOL_VERSION 
(third_party/openssl/boringssl/src/ssl/tls_record.cc:592 
0x7ccce0fd48:0x00000001)
at 
com.google.android.gms.org.conscrypt.NativeCrypto.SSL_do_handshake(Native 
Method)
at 
com.google.android.gms.org.conscrypt.NativeSsl.doHandshake(:com.google.android.gms@200475037@20.04.75 
(120400-295998686):6)
at 
com.google.android.gms.org.conscrypt.ConscryptFileDescriptorSocket.startHandshake(:com.google.android.gms@200475037@20.04.75 
(120400-295998686):14)
... 26 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30100190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Nokia
Device: B2N_sprout
Model: Nokia 7 plus
Id: QKQ1.190828.002
Product: Onyx_00WW

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 00WW_4_10D
```

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky thought this has been fixed with 3.10.1 😳

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky will this be resolved with the 3.11 changes? I guess so.

**Comment by tobiasKaminsky** [comment]
Should be resolved, please try 3.11 via google beta, or wait for release next week.
If it still happen, please re-open.

**Comment by moviuro** [comment]
> Should be resolved, please try 3.11 via google beta, [...]

Everything back in order now!

Still, I'm surprised at how TLS1.3 breaks an app...
